xlators/protocol/client/src/client-rpc-fops.c

```diff
@@ -5087,6 +5092,12 @@ client3_3_lk(call_frame_t *frame, xlator_t *this, void *data)
         goto unwind;
     }
 
+    ret = dict_get_int32(args->xdata, "fix-open", &fix_open);
+    if (!ret && fix_open)
+        local->fix_open = 1;
+    else
+        local->fix_open = 0;
+
```

Comment:
Why not use local->fix_open in line 5095?

Reply:
Done.